suisa_sendemeldung.conf

```diff
@@ -0,0 +1,27 @@
+# Access key for ACRCloud
+#access_key=7cb1cf911d06e66ae2d226926a9012a6
+# Stream Id in ACRCloud
+#stream_id=s-qXuJARB
```

Comment:
I guess we have to reset our credentials :(

Reply:
v2 replaces them anyways :sweat_smile:

Reply:
Good thing :smiling_face_with_tear: